src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -19,15 +22,24 @@ public void canGet() {
 
   @Test
   public void canPutAndGet() {
-    String rule = "priority: t, s, c, b, a, m, g\nfallback-policy: l no-circulation r no-hold n basic-notice o basic-overdue i basic-lost-item\n";
-
-    circulationRulesFixture.updateCirculationRules(rule);
+    UUID lp1 = UUID.randomUUID();
+    UUID lp2 = UUID.randomUUID();
+    UUID rp1 = UUID.randomUUID();
+    UUID rp2 = UUID.randomUUID();
+    UUID np1 = UUID.randomUUID();
+    UUID np2 = UUID.randomUUID();
+    UUID op1 = UUID.randomUUID();
+    UUID op2 = UUID.randomUUID();
+    UUID ip1 = UUID.randomUUID();
+    UUID ip2 = UUID.randomUUID();
+
+    String rule = "priority: t, s, c, b, a, m, g\nfallback-policy: l " + lp1 + " r " + rp1 + " n " + np1 + " o " + op1 + " i " + ip1 + "\n";
+    setRules(rule);
 
     assertThat(getRulesText(), is(rule));
 
-    rule = "priority: t, s, c, b, a, m, g\nfallback-policy: l loan-forever r two-week-hold n two-week-notice o forever-overdue i forever-lost-item\n";
-
-    circulationRulesFixture.updateCirculationRules(rule);
+    rule = "priority: t, s, c, b, a, m, g\nfallback-policy: l " + lp2 + " r " + rp2 + " n " + np2 + " o " + op2 + " i " + ip2 + "\n";
+    setRules(rule);
```

Comment:
I thought this changed stopped rules being saved with policies that do not exist. This test appears to use policy IDs that do not refer to existing policies. What have I missed?